test/parallel/test-fs-existssync-false.js

```diff
@@ -1,12 +1,15 @@
 'use strict';
-
 const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+
+// This test ensures that fs.existsSync doesn't incorrectly return false
+// (especially on Windows)
+// https://github.com/nodejs/node-v0.x-archive/issues/3739
+
 const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
 
-const tmpdir = require('../common/tmpdir');
-
```

Comment:
Nit: I personally would not mode this.

Reply:
Isn't that in conflict with the canonical test structure?

Ref: https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#lines-1-3

Reply:
This is specific to the `require('../common');` but it is fine to keep it as is.